src/core/probe/manager.rs

```diff
@@ -321,9 +299,20 @@ impl ProbeSet {
         })?;
 
         // All probes loaded, issue an info log.
-        info!("{} probe(s) loaded", self.probes.len());
+        info!("{} probe(s) loaded", probes.len());
```

Comment:
nit: I've found it confusing that this line is printed many times. Maybe we could consider moving it to ProbeManager and just printing the total  number of loaded probes?